docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
```

Comment:
I assume you're referring to the second diagram. I took it from https://github.com/mesosphere/kubernetes-mesos/blob/master/docs/README.md